### DISCUSSION TITLE
ScriptReference/TooltipAttribute

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#110] [Notes]
# ScriptReference/TooltipAttribute

https://docs.unity3d.com/ScriptReference/TooltipAttribute.html

**Comment by docscomments** [comment]
**Comment by Jonatan Lemos Zuluaga:**  

`health` has to be `public` in order to appear in the Inspector.

**Comment by andreasIsCoding** [comment]
Note that by default all tool tips are disabled in play mode.  They can be enabled by navigating to: Preferences > General, and checking the “Enable PlayMode Tooltips” box.

Further reading: https://discussions.unity.com/t/regression-profiler-tooltips-no-longer-appear-during-playback/809957
